mtp_cashbook/apps/feedback/forms.py

```diff
@@ -0,0 +1,17 @@
+from django.utils.text import slugify
+from mtp_common.auth.api_client import get_connection
+from mtp_common.auth.exceptions import Unauthorized
+from slumber.exceptions import HttpClientError
+from zendesk_tickets.forms import EmailTicketForm
+
+
+class PrisonTicketForm(EmailTicketForm):
+    def submit_ticket(self, request, subject, tags, ticket_template_name, **kwargs):
+        try:
+            client = get_connection(request)
+            prisons = sorted(map(lambda prison: prison['name'], client.prisons.get()['results']))
+            subject += ' (%s)' % ', '.join(prisons)
```

Comment:
I'd argue that the format string makes the structure more explicit; `+= ' (%s)'`  is ever so slightly harder to parse, and will be problematic if we ever end up translating the interface into Welsh. (Is that a possibility? I have no idea…)

`accessible_prisons` seems like an appropriate name to me, though.